tests/test_mimic_fhir_row_count.sql

```diff
@@ -0,0 +1,39 @@
+    SELECT COUNT(*) AS n_row_observed,   13350281 AS n_row_expected, 'medication_dispense' AS table_name FROM mimic_fhir.medication_dispense UNION ALL
+    SELECT COUNT(*) AS n_row_observed,    1670590 AS n_row_expected, 'medication_dispense_ed' AS table_name FROM mimic_fhir.medication_dispense_ed UNION ALL
+    SELECT COUNT(*) AS n_row_observed,       6461 AS n_row_expected, 'medication_mix' AS table_name FROM mimic_fhir.medication_mix UNION ALL
+    SELECT COUNT(*) AS n_row_observed,   16217713 AS n_row_expected, 'medication_request' AS table_name FROM mimic_fhir.medication_request UNION ALL
+    SELECT COUNT(*) AS n_row_observed,    2733573 AS n_row_expected, 'medication_statement_ed' AS table_name FROM mimic_fhir.medication_statement_ed UNION ALL
+    SELECT COUNT(*) AS n_row_observed,    7477876 AS n_row_expected, 'observation_datetimeevents' AS table_name FROM mimic_fhir.observation_datetimeevents UNION ALL
+    SELECT COUNT(*) AS n_row_observed,    4637088 AS n_row_expected, 'observation_ed' AS table_name FROM mimic_fhir.observation_ed UNION ALL
+    SELECT COUNT(*) AS n_row_observed,     304527 AS n_row_expected, 'observation_micro_org' AS table_name FROM mimic_fhir.observation_micro_org UNION ALL
+    SELECT COUNT(*) AS n_row_observed,    1163623 AS n_row_expected, 'observation_micro_susc' AS table_name FROM mimic_fhir.observation_micro_susc UNION ALL
+    SELECT COUNT(*) AS n_row_observed,    2297709 AS n_row_expected, 'observation_micro_test' AS table_name FROM mimic_fhir.observation_micro_test UNION ALL
+    SELECT COUNT(*) AS n_row_observed,    4450049 AS n_row_expected, 'observation_outputevents' AS table_name FROM mimic_fhir.observation_outputevents UNION ALL
+    SELECT COUNT(*) AS n_row_observed,   10473440 AS n_row_expected, 'observation_vital_signs' AS table_name FROM mimic_fhir.observation_vital_signs UNION ALL
+    SELECT COUNT(*) AS n_row_observed,          1 AS n_row_expected, 'organization' AS table_name FROM mimic_fhir.organization UNION ALL
+    SELECT COUNT(*) AS n_row_observed,     315460 AS n_row_expected, 'patient' AS table_name FROM mimic_fhir.patient UNION ALL
+    SELECT COUNT(*) AS n_row_observed,     704124 AS n_row_expected, 'procedure' AS table_name FROM mimic_fhir.procedure UNION ALL
+    SELECT COUNT(*) AS n_row_observed,    2094688 AS n_row_expected, 'procedure_ed' AS table_name FROM mimic_fhir.procedure_ed UNION ALL
+    SELECT COUNT(*) AS n_row_observed,     731788 AS n_row_expected, 'procedure_icu' AS table_name FROM mimic_fhir.procedure_icu UNION ALL
+    SELECT COUNT(*) AS n_row_observed,    1670188 AS n_row_expected, 'specimen' AS table_name FROM mimic_fhir.specimen UNION ALL
+    SELECT COUNT(*) AS n_row_observed,   14081306 AS n_row_expected, 'specimen_lab' AS table_name FROM mimic_fhir.specimen_lab
+    -- below are the two largest tables by an order of magnitude
+    -- commenting them out reduces query time from ~10 minutes to just ~1 minute
```

Comment:
We either need to comment out labs/chartevnets or add back `UNION ALL` to the specimen_lab to make script run